### PR TITLE
Adding a test to the path /posts/homePagePosts

### PR DESCRIPTION
**Pull request by idantb101**
adding a test for the path - /posts/homePagePosts

### CODE REVIEW

File: tests/server_test.js [hunk 1]
```diff
@@ -9,10 +9,37 @@ test('Testing empty root get request - / ', t => {
     .expect('Content-Type', /json/)
     .expect(200)
     .end((err, res) => {
-        console.log('in end');
-        console.log(res.text)
+      console.log(res.text);
```
Comment (Amirk390) [inline tests/server_test.js]: We should avoid using console.logs

if you'll look at #17 i remove all the console.logs at the other files
we should find other way to get errors

**Comment by idantb101** [comment]
Adding a new path for all tags and testing it

File: tests/server_test.js [hunk 2]
```diff
@@ -10,6 +10,53 @@ test('Testing empty root get request - / ', t => {
...
+    }
+  ];
+  request(app)
+    .get('/homePagePosts')
+    .expect('Content-Type', /json/)
+    .expect(200)
+    .end((err, res) => {
+      t.ok(res.text);
+      let actualThings = res.body.homePagePosts;
+      console.log(actualThings);
```
Comment (Amirk390) [inline tests/server_test.js]: please change this line to 
please remove the console.logs or tag them with TODO for later fixes

**Comment by idantb101** [comment]
Tanks !!!

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
     "start": "node src/app.js",
     "test": "node tests/server_test.js",
```
Comment (claireinez) [inline package.json]: Fix your test script to run all the tests

Comment (harrygfox) [reply package.json]: @superhealthyminds/developers can we confirm that this has been completed? Or is there a plan to complete it at a later date?

**Comment by harrygfox** [comment]
@superhealthyminds/developers the comment above from @claireinez that reads:
> Fix your test script to run all the tests
Is this something that can be done in a new PR?

**Pull request closed without merging**